modules/jupyter-widget/src/utils.js

```diff
@@ -89,24 +89,39 @@ function getTooltip(pickedInfo) {
 }
 
 function tabularize(json) {
+  // Turns a JSON object of picked info into HTML for a tooltip
   const dataTable = document.createElement('div');
   dataTable.className = 'dataTable';
 
+  // Creates rows of two columns for the tooltip
   for (const key in json) {
+    if (['undefined', 'position', 'index'].includes(key)) {
+      continue; // eslint-disable-line
+    }
     const row = document.createElement('div');
     const header = document.createElement('div');
     header.className = 'header';
     header.innerText = key;
     Object.assign(header.style, {
-      fontWeight: 500,
-      marginRight: '10px'
+      fontWeight: 700,
+      marginRight: '10px',
+      flex: 1
     });
     const value = document.createElement('div');
     value.className = 'value';
-    value.innerText = JSON.stringify(json[key]);
+
+    if (Array.isArray(json[key])) {
```

Comment:
I think it's still valuable to see the array content when it's short, for example `color: [255, 0, 0]`. Maybe check if the array is less than 4 element long and the first element is a number?